specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/virtualNetwork.json

```diff
@@ -2089,6 +2094,18 @@
       },
       "description": "DhcpOptions contains an array of DNS servers available to VMs deployed in the virtual network. Standard DHCP option for a subnet overrides VNET DHCP options."
     },
+    "PrivateEndpointVNetPolicies": {
+      "type": "string",
+      "description": "Private Endpoint VNet Policies.",
```

Comment:
Is this being used for anything else other than pl64k item? Is it worth mentioning in description in that case?

Reply:
This setting controls enablement for PL64K. This is the only planned use for this property.